packages/cli/lib/services/__snapshots__/model.service.unit.test.ts.snap

```diff
@@ -202,11 +203,20 @@ interface TableauCredentials extends CredentialsCommon {
     token?: string;
     expires_at?: Date | undefined;
 }
+interface JWTCredentials {
+    type: AuthModes['Jwt'];
+    api_key?: string;
```

Comment:
can we add a comment to make it clear that `api_key` is to support provider like Ghost that use a composite api_key to pass the id and the privateKeyId and privateKey?

Maybe we can even make the type more explicit to make it clear it is either `api_key` or `privateKey/privateKeyId`. Ex:
```
interface JWTCredentials {
    type: AuthModes['Jwt'];
    privateKey: {
       id: string,
       secret: string
    } | string, // colon separated string: 'id:secret'
    issuerId?: string;
    token?: string;
    expires_at?: Date | undefined;
``` 
We would also need to make the `/auth/jwt` endpoint match this type if possible

Reply:
Yes, I have updated this, making `privateKey` required since it can be of `string` or an object with `id` and `secret`. Please have a look and let me know.